tests/test-codex-hook-fixtures.sh

```
@@ -1029,6 +1029,140 @@ EOF
+      -c "hooks.Stop=$override" \
+      -o "$result2" \
+      - >/dev/null 2>"$stderr2" || rc2=$?
+
+  case "$rc2" in
+    0|124|137)
+      # 0=정상, 124=SIGTERM-by-timeout, 137=SIGKILL-by-timeout — 모두 supervisor가 정리한 PASS.
+      # 잔존 codex/timeout 프로세스가 sandbox path로 식별되는지 확인 (process group kill 입증).
+      # parallel-audit A fix: macOS pgrep -fc/-fa 미지원 → portable ps + grep -F (fixed string).
+      sleep 1  # SIGKILL grace 후 OS reaper에 시간 부여
+      local lingering_pids lingering_count lingering_lines
+      lingering_pids=$(ps -axo pid=,command= 2>/dev/null | grep -F -- "$sandbox_path" | grep -v "^[[:space:]]*${self_pid}[[:space:]]" | grep -v 'grep -F -- ' | awk '{print $1}' || true)
```

Shellcheck (0.11.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)